src/components/Comments.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import { getCommentsByArticle } from "../api";
+import { useParams } from "react-router-dom";
+
+export const Comments = () => {
+  const params = useParams();
+  const [comments, setComments] = useState([]);
+  const [loadingComments, setLoadingComments] = useState(true);
+
+  useEffect(() => {
+    setLoadingComments(true);
+    getCommentsByArticle(params.article_id).then((data) => {
+      setComments(data.comments);
+      setLoadingComments(false);
+    });
+  }, [params.article_id]);
+
+  if (!comments) {
+    return null;
```

Comment:
Good to consider this here, as there may be a situation where an article has no comments. But instead of null, could we return something a bit more descriptive to explain to the user what has happened?